Cabal/Cabal.cabal

```diff
@@ -1,5 +1,5 @@
 name:          Cabal
-version:       1.25.0.0
+version:       2.0.0.0
```

Comment:
This is usually only done when the release branch is created. If you want to disambiguate between your branch and `master`, I suggest using something like `1.25.1.0`.